core/dbt/contracts/graph/model_config.py

```diff
@@ -335,6 +335,15 @@ def replace(self, **kwargs):
 @dataclass
 class SourceConfig(BaseConfig):
     enabled: bool = True
+    quoting: Optional[Dict[str, Any]] = None
+    freshness: Optional[Dict[str, Any]] = None
+    loader: Optional[str] = None
+    # TODO what type is this? docs say: "<column_name_or_expression>"
```

Comment:
A string.  It's literally the name of the column that holds the `loaded_at` time.  It's also defined as a string [here](https://github.com/dbt-labs/dbt-core/blob/7f953a6d48a4db254b17fd3929d267e4b6069228/core/dbt/contracts/graph/parsed.py#L629)